recipes-bsp/u-boot/files/0001-Remove-video-config.patch

```diff
@@ -0,0 +1,24 @@
+From 7ff28a5520475c207b67c04b05fe32d42461215e Mon Sep 17 00:00:00 2001
+From: "Rodrigo M. Duarte" <rodrigo.duarte@ossystems.com.br>
+Date: Mon, 16 Jan 2023 15:05:00 -0300
+Subject: [PATCH] Remove video config
+
+---
+ configs/sandbox_defconfig | 1 -
```

Comment:
I believe we should use the `tools-only_defconfig` so we avoid adding this extra patch.